Although stated in the manpage the new version does not use ~/Maildir if
it exis but asks for the maildir instead. Looking at the code I
saw that one check was inverted. The following patch fixes this.

Michael Guntsche <mike@it-loops.com>
---
 src/mu-util.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: maildir-utils/src/mu-util.c
===================================================================
--- maildir-utils.orig/src/mu-util.c	2010-01-22 11:50:54.000000000 +0900
+++ maildir-utils/src/mu-util.c	2010-01-22 11:51:04.000000000 +0900
@@ -91,7 +91,7 @@
 
 	/* then, try ~/Maildir */
 	dir = mu_util_dir_expand ("~/Maildir");
-	if (!mu_util_check_dir (dir, TRUE, FALSE))
+	if (mu_util_check_dir (dir, TRUE, FALSE))
 		return dir;
 
 	/* nope; nothing found */
